### ISSUE TITLE
Can't connect to server (Win11)

### ISSUE THREAD

**Issue by IlyaBezus** [Jigsaw-Code/outline-client#1383]
**Describe the bug**
I'm trying to connect to my DigitalOcean server, and it does not work. Label "Connecting..." pops up, and then it's back to "Disconnected". Although this connection does work on my mobile phone. Sometimes, there is an error below: "we can't seem to connect to your server". I tried to run both client and installer as administrator.

**To Reproduce**
Fresh install, win11.

**Expected behavior**
It should connect :)

**Screenshots**
![Untitled](https://user-images.githubusercontent.com/14222481/184551330-36826ff3-6adb-4ce2-81ef-c5fdbdfed264.png)
![Untitled2](https://user-images.githubusercontent.com/14222481/184551363-84f3b2b0-b038-44f2-9fad-c9de119e784b.png)

**Client System (please complete the following information):**
 - Outline Client Version: Win 1.7.1
 - Your Operating System: Win 11

**Comment by outline-support** [comment]
Thank you for letting us know! We will share this issue with our product team to investigate further. If you have additional questions or would like to share more information about this issue, please contact us at support.getoutline.org.

**Comment by MintBlitz** [comment]
No problems on my side, it's working perfectly on Windows 11 with Outline Windows Client 1.7.1 and Server 1.6.1
Edition	Windows 11 Pro
Version	21H2
OS build	22000.856
Experience	Windows Feature Experience Pack 1000.22000.856.0

**Comment by IlyaBezus** [comment]
> Version 21H2
> OS build 22000.856
> Experience Windows Feature Experience Pack 1000.22000.856.0

Yeah, same version exactly, wonder why it does not work for me :(

**Comment by jean-louis1776** [comment]
> > Version 21H2
> > OS build 22000.856
> > Experience Windows Feature Experience Pack 1000.22000.856.0
> 
> Yeah, same version exactly, wonder why it does not work for me :(

It's working! bug fixed

**Comment by MintBlitz** [comment]
> > > Version 21H2
> > > OS build 22000.856
> > > Experience Windows Feature Experience Pack 1000.22000.856.0
> > 
> > 
> > Yeah, same version exactly, wonder why it does not work for me :(
> 
> It's working! bug fixed

What did you do to fix the problem

**Comment by jean-louis1776** [comment]
> > > > Version 21H2
> > > > OS build 22000.856
> > > > Experience Windows Feature Experience Pack 1000.22000.856.0
> > > 
> > > 
> > > Yeah, same version exactly, wonder why it does not work for me :(
> > 
> > 
> > It's working! bug fixed
> 
> What did you do to fix the problem

Nothing...the program just started working again as before

**Comment by lolmaus** [comment]
I'm having the same issue: the Outline client on Windows simply refuses to connect:

> We can't seem to connect to your server. Please check that you are connected to the internet and try again.

It disconnects immeditely after clicking Connect, but the error message appears with the delay.

There are a few other apps that stopped working, e. g. GeForce Experience.

Rebooting, reinstalling the client do not help.

Windows 10 Pro Version	10.0.19044 Build 19044
Outline client 1.7.1

Please help.

**Comment by lolmaus** [comment]
My issue seems to have been caused by Surfshark VPN: uninstalling Surfshark fixed Outline Client and GeForce Experience.

**Comment by jepbura** [comment]
Hi
I had the same problem, Disabling the Hyper-V solved my problem.

You can disable Hyper-V using the Windows Features dialog:

1- Press Win + R to open Run.
2- Type control and click OK to open the Control Panel.
3- In the Control Panel, go to Programs > Programs and Features.
4- In the left pane, click on Turn Windows features on or off.
5- In the Windows Features dialog, deselect Hyper-V.
6- Click OK.
7- Once completed, click on Restart Now to restart, and run outline again.

![Screenshot 2022-10-11 120420](https://user-images.githubusercontent.com/25738325/195041144-0fd7b2fb-0597-4dac-b3b6-c7d629bbbd83.png)

**Comment by nikulov** [comment]
I'm having the same issue again, any solutions?
